internal/install/recipes/recipe_filter_runner.go

```diff
@@ -73,6 +73,7 @@ func (rf *RecipeFilterRunner) RunFilterAll(ctx context.Context, r []types.OpenIn
 		filtered := rf.RunFilter(ctx, &recipe, m)
 
 		if !filtered {
+			rf.installStatus.RecipeCompatilityDetected(recipe)
```

Comment:
I'm not sure this implements the logic defined in the story, but one way to validate would be to add the tests that cover the story ACs, I've also put some comment in the recipe_installer_test.go file

Reply:
I'll double check, but after our conversation I think this is where we wanted it. I'll add the test mentioned above to verify this is where we want this called.